src/server/game/Movement/MotionMaster.cpp

```diff
@@ -382,6 +382,8 @@ void MotionMaster::MoveJump(float x, float y, float z, float speedXY, float spee
     init.MoveTo(x, y, z, false);
     init.SetParabolic(max_height, 0);
     init.SetVelocity(speedXY);
+    if (HasDst)
```

Comment:
What is your reasoning behind using a bool flag instead of [`std::nanf`](http://www.cplusplus.com/reference/cmath/nanf/) or similar non-numeric float constants?